blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```

Comment:
You'll want the meta tag referenced in week one on all of your HTML pages

Reply:
I'm not sure which tag you are referring to. Do you mean `<meta name="viewport" content="width=device-width, initial-scale=1">`

Is it week one of HTML200? Was it referenced in a video? any help would be appreciated